src/pages/blog/components/_hero.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { TextWrapper, Title } from './_common';
+import device from 'themes/device.js'
+import { Flex } from 'components/containers'
+import { QueryImage } from 'components/elements'
+
+const MainWrapper = styled.div`
+    width: 100%;
+    background: black;
+    position: relative;
+    height: 600px;
+
+    .wrap {
+        max-width: 1440px;
+    }
+    .hero-content{
+        position: relative;
+        z-index: 99;
+    }
+
+    @media ${device.tablet} {
+        height: 400px;
+        text-align: left;
+        object-fit: cover;
+    }
+`
```

Comment:
I don't think we're using this as well 🙏  Rest LGTM

Reply:
Thanks for the review. Only the responsive part is used